sqldelight-gradle-plugin/src/test/kotlin/com/squareup/sqldelight/FixturesTest.kt

```diff
@@ -60,6 +60,21 @@ class FixturesTest {
       return
     }
 
+    var removeGradleAfter = false
+    var removeManifestAfter = false
```

Comment:
Make these fields, put the copy logic in a `@Before` method, and put the delete logic in an `@After` method. Currently if the test fails they won't be deleted.